### ISSUE TITLE
Argument passed partially with brackets

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#429]
**Describe the bug**
The below code results in 3 errors.

**To Reproduce**
```
Public Sub Main()

    Dim Var As Long
    Test (1&) Or 1&
    Test 1& Or 2& ' This is OK, no errors.
    
End Sub

Private Sub Test(ByRef Lng As Long)
    Debug.Print Lng
End Sub
```

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/133072562-d3a01008-8b13-4cfb-850c-dd40167e28c9.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.5788]

**Additional context**
The Test method can also be ByVal. That does not matter.

**Comment by wqweto** [comment]
This must be a side-effect of `Call` keyword being optional in TB i.e. `Call MyFunc(1)` can be shortened to `MyFunc(1)` as in any other sane language.

Unfortunately now your `Test(1) Or 1` expression reads like first calling `Test` function with `1` for actual argument and then `Or` becomes superfluous for the compiler.

**Comment by WaynePhillipsEA** [comment]
Yes, indeed, it's an error in the parser treatment of this case due to the changes allowing for optional Call prefix.  

It is easily fixed, but I'm a little busy finalizing the GUI framework at the minute.  It will be fixed by the end of this week.
